tests/tests.py

```diff
@@ -67,3 +67,8 @@ def test_suite():
     suite.addTest(MountedDataDirectoryTest(docker_layer))
     suite.layer = docker_layer
     return suite
+
+
+if __name__ == "__main__":
+    testRunner = unittest.TextTestRunner()
+    testRunner.run(test_suite())
```

Comment:
Might not be necessary if `test_suite` is adapted to the [load_tests protocol](https://docs.python.org/3/library/unittest.html?highlight=unittest#load-tests-protocol)

Reply:
cool, but to us non, strictly speaking, pythonistas, the explicitness of having a __main__ removes all the magic, for me this is a simpler, easier approach.

Reply:
And for pythonistas it is odd that is doesn't work the "standard" way. Could we change it to have both?

Reply:
:)